src/ctia/stores/es/crud.clj

```diff
@@ -453,6 +453,18 @@ It returns the documents with full hits meta data including the real index in wh
                                            access-control-filter-list
                                            ident
                                            get-in-config))))))
+(s/defn rename-search-fields :- (s/maybe {s/Keyword [s/Str]})
+  "Automatically translates keyword fields to use underlying text field.
+
+   ES doesn't like when different types of tokens get used in the same query. To deal with
+   that, we create a nested field of type 'text', see:
+   `ctia.stores.es.mapping/searchable-token`. This should be opaque - caller shouldn't
+   have to explicitly instruct API to direct query to the nested field."
+  [fields :- (s/maybe [s/Any])]
+  (let [search-fields-mapping {"source" "source.text"}]
```

Comment:
you need to include all fields which are in default search and mapped as searchable-token: https://github.com/threatgrid/ctia/pull/1155/files